rte/main.go

```diff
@@ -96,7 +96,11 @@ func main() {
 
 	// TODO: recycled flag (no big deal, but still)
 	cli = podexclude.NewFromLister(cli, parsedArgs.RTE.Debug, parsedArgs.LocalArgs.PodExcludes)
-	err = resourcetopologyexporter.Execute(cli, parsedArgs.NRTupdater, parsedArgs.Resourcemonitor, parsedArgs.RTE)
+	hnd := resourcemonitor.Handle{
+		PodResCli: cli,
+		K8SCli:    k8scli,
```

Comment:
I see k8cli is defined only in a later [commit](https://github.com/openshift-kni/numaresources-operator/pull/772/commits/6143f74d62b16860806c0216924e4de3cf0731d3#diff-fa3097e800fda70b00ec99071e568704f2584b6b7759c8f6149628535e90a8ceR85).

Reply:
fixed the ordering, thanks!